src/EPR.Calculator.API/Builder/Summary/CalcResultSummaryBuilder.cs

```diff
@@ -279,6 +310,31 @@ private static CalcResultSummaryProducerDisposalFees GetProducerRow(
             };
         }
 
+        //section bad debt total
```

Comment:
Please move all new functions to a new class
Also, create a subfolder for this class and call it OnePlus2A

Reply:
Resolving it as per discussion in call